src/test/java/uk/gov/ons/ctp/response/action/export/service/TemplateServiceIT.java

```diff
@@ -266,12 +158,28 @@ private String getLatestSftpFileName() throws IOException {
 
     String sftpPath = "Documents/sftp/";
     ChannelSftp.LsEntry[] sftpList = defaultSftpSessionFactory.getSession().list(sftpPath);
+
+    System.out.println("Latest File Length list: " + sftpList.length);
```

Comment:
Is it worth putting this into a log?

Reply:
Probably best to just remove, I'll do that